extensions/socketio/package.json

```diff
@@ -30,6 +28,10 @@
     "lodash": "^4.17.20",
     "socket.io": "^2.3.0"
   },
+  "peerDependencies": {
+    "@loopback/boot": "^3.1.0",
+    "@loopback/core": "^2.12.0"
```

Comment:
We also need to add both of them to `devDependencies` as `npm 6.x` does not install peer dependencies.